presto-parser/src/main/java/io/prestosql/sql/tree/FunctionCall.java

```diff
@@ -48,6 +49,7 @@ public FunctionCall(
             Optional<Expression> filter,
             Optional<OrderBy> orderBy,
             boolean distinct,
+            boolean ignoreNulls,
             List<Expression> arguments)
```

Comment:
Please update `equals()` and `hashCode()` to take account of the new field `ignoreNulls`.